pkg/platform/local/platform_test.go

```diff
@@ -126,6 +126,59 @@ func (suite *TestSuite) TestValidateFunctionContainersHealthiness() {
 	suite.Require().Equal(function.GetStatus().State, functionconfig.FunctionStateError)
 }
 
+// Test function import without deploy and build, then deploy calls build and deploy
+func (suite *TestSuite) TestImportFunctionFlow() {
```

Comment:
End-to-end export and import flow on local platform?
Same for projects

Reply:
How can I do that in a platform test suite, if export is done only in the dashboard?

Reply:
Crap you're right I assumed they were e2e
We're missing a place to put those